Frames.podspec

```diff
@@ -15,8 +15,15 @@ Pod::Spec.new do |s|
 
   s.source_files = 'Source/**/*.swift'
   s.exclude_files = "Classes/Exclude"
-  s.resource_bundles = {
-    'FramesIos' => 'Source/Resources/**/*'
-  }
+  s.resources = 'Source/Resources/**/*'
 
+  s.dependency 'PhoneNumberKit', '~> 3.3.0'
+  s.dependency 'Alamofire', '~> 5.4.0'
```

Comment:
we can get rid of this as we dont use it.

Reply:
no it is in use